src/evm/update_error_handling.sh

```
@@ -0,0 +1,52 @@
+    "block.zig"
+    "comparison.zig"
+    "control.zig"
+    "crypto.zig"
+    "environment.zig"
+    "log.zig"
+    "memory.zig"
+    "stack.zig"
+    "system.zig"
+)
+
+cd opcodes
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)